pinot-connectors/prestodb-pinot-dependencies/pinot-core-jdk8/pom.xml

```diff
@@ -112,8 +112,10 @@
     </dependency>-->
     <dependency>
       <groupId>io.netty</groupId>
-      <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>${netty-tcnative.version}</version>
+      <artifactId>netty-bom</artifactId>
+      <version>${netty.version}</version>
+      <type>pom</type>
+      <scope>import</scope>
```

Comment:
shouldn't we define this in the dependencyManagement section? since it is import scope? 
after this it gave warning in maven:
```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.apache.pinot:pinot-core:jar:0.11.0-SNAPSHOT
[WARNING] 'dependencies.dependency.scope' for io.netty:netty-bom:pom must be one of [provided, compile, runtime, test, system] but is 'import'. @ line 130, column 14
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.apache.pinot:pinot-pulsar:jar:0.11.0-SNAPSHOT
[WARNING] 'dependencies.dependency.scope' for io.netty:netty-bom:pom must be one of [provided, compile, runtime, test, system] but is 'import'. @ line 310, column 14
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```

Reply:
anyway I created https://github.com/apache/pinot/pull/8495 plz kindly take a look